NitroxModel/Packets/FootstepPacket.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+
+namespace NitroxModel.Packets
+{
+    [Serializable]
+    public class FootstepPacket : Packet
+    {
+        public ushort playerID { get; }
+        public StepSounds assetIndex { get; }
```

Comment:
public fields/properties should be PascalCase

Reply:
Done